src/www/widgets/widgets/carp_status.widget.php

```diff
@@ -54,13 +54,13 @@
 <?php
       if (get_single_sysctl('net.inet.carp.allow') <= 0 ) {
           $status = gettext("DISABLED");
-          echo "<span class=\"fa fa-remove fa-fw text-danger\" title=\"$status\" alt=\"$status\" ></span>";
+          echo "<span class=\"fa fa-remove fa-fw text-danger\" title=\"$status\" ></span>";
```

Comment:
does this still display tooltips?

Reply:
`title` is the attribute that displays tooltips. `alt` is only used when an img(!) cannot load (except in IE, which interprets the `alt` as `title` if no `title` is specified).